filenamify.js

```diff
@@ -58,3 +58,5 @@ export default function filenamify(string, options = {}) {
 
 	return string;
 }
+
+const trimRepeatedReservedChars = string => string.replace(reRepeatedReservedChars, '$1');
```

Comment:
```suggestion
const trimRepeatedReservedCharacters = string => string.replace(reRepeatedReservedCharacters, '$1');
```

Reply:
I'm thinking about completely removing `const trimRepeatedReservedChars`, given that after moving the regex inside `filenamify()`, calling `string.replace(...)` inside the if statement would be the easiest way to do it.